.github/workflows/docs.yml

```diff
@@ -28,15 +28,10 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.x'
-
       - name: Set up Pixi
-        uses: prefix-dev/setup-pixi@v0.8.10
+        uses: prefix-dev/setup-pixi@v0.9.1
         with:
-          pixi-version: v0.50.2
+          pixi-version: v0.56.0
```

Comment:
Might as well consistently use the latest version.

Reply:
It is unclear to me if `latest` would work as a version requirement here. It seems like [`pixi-url` accepts such a version requirement](https://github.com/prefix-dev/setup-pixi?tab=readme-ov-file#download-pixi-from-a-custom-url), so maybe?